ieeeuoft/hackathon-template
# Installed Flow and replaced node-sass with sass in Dashboard App

## Overview

- Adds [Flow](https://flow.org/en/docs/getting-started/) to Dashboard app so we can type API responses
- Removes `node-sass `and replaces with `sass` (Was having yarn issues with Flow before doing this)


## Unit Tests Created

N/A


## Steps to QA

- In the hackathon_site/dashboard/frontend directory, run:
`yarn install --frozen-lockfile`
`yarn flow`

This will start the Flow server and `No errors!` should return since none of our files have been set up with Flow yet.
